libs/types/src/election.ts

```diff
@@ -724,6 +724,10 @@ export type YesNoVote =
   | readonly ['no', 'yes']
   | readonly [];
 export type YesOrNo = Exclude<YesNoVote[0] | YesNoVote[1], undefined>;
+export const DisplayTextForYesOrNo: Record<YesOrNo, string> = {
+  yes: 'Yes',
+  no: 'No',
+};
```

Comment:
The display text for yes/no votes seems more like a UI concern than a typing concern. Is there a place this would make sense to live in the ui library?

Reply:
Good call! It could probably go in [`libs/ui/src/globals.ts`](https://github.com/votingworks/vxsuite/blob/main/libs/ui/src/globals.ts) instead